-------------------------------------------------------------------------------
Test set: CipherKitTest
-------------------------------------------------------------------------------
Tests run: 31, Failures: 2, Errors: 0, Skipped: 0, Time elapsed: 0.808 s <<< FAILURE! -- in CipherKitTest
CipherKitTest.testNonceGenerationNegativeSize -- Time elapsed: 0.041 s <<< FAILURE!
java.lang.AssertionError: 

Expecting actual throwable to be an instance of:
  java.lang.IllegalArgumentException
but was:
  java.lang.NegativeArraySizeException: -1
	at Utilities.CipherKit.generateNonce(CipherKit.java:100)
	at CipherKitTest.lambda$testNonceGenerationNegativeSize$0(CipherKitTest.java:63)
	at org.assertj.core.api.ThrowableAssert.catchThrowable(ThrowableAssert.java:63)
	...(75 remaining lines not displayed - this can be changed with Assertions.setMaxStackTraceElementsDisplayed)
	at CipherKitTest.testNonceGenerationNegativeSize(CipherKitTest.java:64)
	at java.base/java.lang.reflect.Method.invoke(Method.java:569)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)

CipherKitTest.testCipherKitCreationEmptyPassword -- Time elapsed: 0.016 s <<< FAILURE!
java.lang.AssertionError: 

Expecting code to raise a throwable.
	at CipherKitTest.testCipherKitCreationEmptyPassword(CipherKitTest.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:569)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)

